README.md

```diff
@@ -57,10 +59,10 @@ jobs:
 
 | parameter | description |
 | - | - |
-| paths_list | List of changed paths, example: `["dist", "dist/data"]` |
-| file_list | List of changed files, example: `["dist/main.tf", "dist/index.js"]` |
-| paths_str | List of changed paths separated by `,` example: `"dist,dist/data"` |
-| file_str | List of changed files separated by `,` example: `"dist/main.tf,dist/index.js"` |
+| `paths_list` | List of changed paths, example: `["dist", "dist/data"]`             |
+| `file_list`  | List of changed files, example: `["dist/main.tf", "dist/index.js"]` |
+| `paths_str`  | List of changed paths separated by `,` example: `"dist,dist/data"`  |
+| `file_str`   | List of changed files separated by `,` example: `"dist/main.tf,dist/index.js"` |
```

Comment:
These changes were generated with some tool? I'm using [action-docs](https://github.com/npalm/action-docs)

Reply:
I don't remember using a tool.  It's possible that Emacs Markdown mode auto-formats this way, but I don't think so; definitely nothing more automated than that.